events/janus_wsevh.c

```diff
@@ -43,6 +43,15 @@ const char *janus_wsevh_get_package(void);
 void janus_wsevh_incoming_event(json_t *event);
 json_t *janus_wsevh_handle_request(json_t *request);
 
+/* Forward declarations */
+void janus_wsevh_schedule_connect_attempt(void);
+void janus_wsevh_calculate_reconnect_delay_on_fail(void);
```

Comment:
This group of declarations should be made _static_.
Also IMHO it should be moved under this block
```c
/* Useful stuff */
static volatile gint initialized = 0, stopping = 0;
static GThread *ws_thread, *handler_thread;
static void *janus_wsevh_thread(void *data);
static void *janus_wsevh_handler(void *data);
```
and the comment should be more informative like e.g.
```c
/* Connection helpers */
```

Reply:
like /* Useful stuff */ 😂 adopting and moving code